_posts/en/2022-01-26-build-website-with-nextjs.md

```diff
@@ -0,0 +1,365 @@
+          </div>
+        </Card>
+        <Card backgroundColorClass="red-background" imagePath="/pandas-logo.png" imageAlt="Eleven Labs Pandas Team" title="Donut Factory">
+          <div className={styles["team-description"]}>
+            <div className={styles["team-description__txt"]}>Pandas who made some donuts from the Donuts planet.</div>
+            <Button className="red-background" onClick={(e) => handleTeamPageClick(e, "donuts-factory")}>READ MORE</Button>
+          </div>
+        </Card>
+        <Card backgroundColorClass="yellow-background" imagePath="/racoon-logo.png" imageAlt="Eleven Labs Racoon Team" title="Raccoons of Asgard">
+          <div className={styles["team-description"]}>
+            <div className={styles["team-description__txt"]}>Racoons guardian of the galaxy from the great Asgard planet.</div>
+            <Button className="yellow-background" onClick={(e) => handleTeamPageClick(e, "asgard-racoons")}>READ MORE</Button>
+          </div>
+        </Card>
+      </FlexContainer>
+    </>
+  )
+}
+```
+
+> **Note**: don't forget to add the CSS rules. They are in the `src/styles/Home.module.css. All CSS modules for pages are in this folder.
+
+![Website Home Page](assets/2022-01-26-build-website-with-nextjs/homepage.png)
+
+The index.js and Layout.js components contain redirections to other website pages, but they don't exist yet. Let's learn how to create these pages NextJS.
+
+### Create a simple static page
+
+First, we need a static page for contact. Static means the page/container doesn't receive any parameter. To create a `/contact` page, add a contact.js file with a React component returning an empty div. NextJS router will automatically create the page:
+
+```shell
+touch src/pages/contact.js
+```
+
+`src/pages/contact.js`:
+```js
+// Components
+import { PageCard } from "../components/PageCard"
+import { FlexContainer, ORIENTATION } from "../components/FlexContainer"
+
+export default function Contact() {
+    return <>
+    </>
+}
+```
+
+Now we have an empty page for `/contact`. Just add the content and, it is done:
+
+```js
+// Components
+import { PageCard } from "../components/PageCard"
+import { FlexContainer, ORIENTATION } from "../components/FlexContainer"
+
+export default function Contact() {
+    return <>
+        <h2>Contact Eleven Labs</h2>
+        <PageCard>
+            <FlexContainer orientation={ORIENTATION.VERTICAL}>
+                <h3>Eleven Labs</h3>
+                <div>PARIS</div>
+                <div>15, avenue de la Grande Armée</div>
+                <div>75116 PARIS</div>
+                <div><a href="mailto:contact@eleven-labs.com">contact@eleven-labs.com</a></div>
+            </FlexContainer>
+        </PageCard>
+    </>
+}
+```
+
+![Contact Page](assets/2022-01-26-build-website-with-nextjs/contactpage.png)
+
+Now we have a contact page. The website needs one page for each astronaut's team. To do that, create a route with the team name as a parameter.
```

Comment:
Same as before, I had the feeling that I had to do it by myself, although you're going to go through that after. I think it is clearer like this :)

```suggestion
Now we have a contact page, the website needs one page for each astronaut's team. To do that, we will create a route with the team name as a parameter.
```